pkg/encoding/kzgEncoder/encoder.go

```diff
@@ -205,21 +205,22 @@ func (g *KzgEncoderGroup) newKzgEncoder(params rs.EncodingParams) (*KzgEncoder,
 }
 
 // just a wrapper to take bytes not Fr Element
-func (g *KzgEncoder) EncodeBytes(inputBytes []byte) (*bls.G1Point, *bls.G1Point, []Frame, []uint32, error) {
+func (g *KzgEncoder) EncodeBytes(inputBytes []byte) (*bls.G1Point, *bls.G2Point, *bls.G2Point, []Frame, []uint32, error) {
 	inputFr := rs.ToFrArray(inputBytes)
 	return g.Encode(inputFr)
 }
 
-func (g *KzgEncoder) Encode(inputFr []bls.Fr) (*bls.G1Point, *bls.G1Point, []Frame, []uint32, error) {
+func (g *KzgEncoder) Encode(inputFr []bls.Fr) (*bls.G1Point, *bls.G2Point, *bls.G2Point, []Frame, []uint32, error) {
 
 	startTime := time.Now()
 	poly, frames, indices, err := g.Encoder.Encode(inputFr)
 	if err != nil {
-		return nil, nil, nil, nil, err
+		return nil, nil, nil, nil, nil, err
 	}
 
 	// compute commit for the full poly
 	commit := g.Commit(poly.Coeffs)
+	lowDegreeCommitment := bls.LinCombG2(g.Srs.G2[:len(poly.Coeffs)], poly.Coeffs)
```

Comment:
How does the verifier know this is committing to the same polynomial as the above commitment (just use a different group)?